views/home.tmpl

```diff
@@ -245,7 +245,9 @@
                                 </span>
                             </div>
                             <div class="fs12 lh1rem text-black-50">
-                                <span data-target="homepage.targetPct">{{printf "%.2f" .PoolInfo.PercentTarget}}</span> % of target&nbsp;<span>{{intComma .PoolInfo.Target}}</span>
+                                <span data-target="homepage.targetPct">{{printf "%.2f" $.PercentChange}}%</span>
+                                {{if lt $.PercentChange 0.0}} under {{else}} over {{end}}target&nbsp;
```

Comment:
Since this `x% over/under target` is the case going forward then on the UI the value `%x` should always be positive.
I think part of this implementation could be moved to the golang side so it can be easy to use `math.Abs(` function. 
 Alternatively the `math.Abs(` implementation could be added to the template.go file and invoked here.

Reply:
Actually in `func makeTemplateFuncMap(params *chaincfg.Params) template.FuncMap {` in template.go adding 
```go
"ToAbsValue": math.Abs,
```
then doing this:
should fix the issue,

` <span data-target="homepage.targetPct">{{printf "%.2f" ToAbsValue($.PercentChange)}}%</span>`

Reply:
@dmigwi Just to be sure I understand, you're saying to just generate the entire string in `explorer.Home`, rather than trying to parse things in the template?

Reply:
That was option 1, the option 2 was to add a `math.Abs` function in the template.go file.

Reply:
Oh. Sorry. Didn't see your second comment before. Yeah, I think either of those would be fine.

Reply:
@buck54321 I understand this changing the current

`-0.3% under target` to `0.3% under target` hence the comment by @dmigwi to make the `%x`

an absolute

Reply:
That's sounds right.